### DISCUSSION TITLE
How do I display instructions in a recipe

### DISCUSSION THREAD

**Discussion by ShuiHuo** [TandoorRecipes/recipes#1382] [Q&A] [answered]
![swappy-20220120-150119](https://user-images.githubusercontent.com/7471938/150289560-31c28f8f-e8f7-47be-b05b-713c4439dcf3.png)

As shown, I want to display the instructions in the recipe view.

Or say... Did I misunderstand these features?

<details>
    <summary>↑↑ from the translation software↑↑↓↓click here to view the original Chinese↓↓</summary>
如图，我想让说明显示在菜谱显示界面。

还是说……我理解错了这些功能。
</details>

**Comment by vabene1111** [comment, marked as answer]
this should work, i am not sure why its not. Can you post a a proper issue with information about your setup ?

**Reply by ShuiHuo** [reply]
I seem to have found the problem. I use nginxconfig.io generates a configuration,
It comes with `security.conf` file,
which contains  
`add_header Content-Security-Policy   "default-src 'self' http: https: data: blob: 'unsafe-inline'; frame-ancestors 'self';" always;` 
Comment out this sentence to display the description.
![swappy-20220121-150209](https://user-images.githubusercontent.com/7471938/150481563-060cc25d-ee62-414d-b126-54da2db1d6e2.png)


<details>
    <summary></summary>
我好像找到问题了，我使用 nginxconfig.io 生成配置，它带有 security.conf 文件，里面包含 add_header Content-Security-Policy   "default-src 'self' http: https: data: blob: 'unsafe-inline'; frame-ancestors 'self';" always; ，把这句话注释掉就能显示说明了。
</details>

**Comment by vabene1111** [comment]
perfect !
